frontend/src/layout/navigation-3000/components/ProjectTree/projectTreeLogic.tsx

```diff
@@ -308,5 +390,20 @@ export const projectTreeLogic = kea<projectTreeLogicType>([
         actions.loadUnfiledItems('insight')
         actions.loadUnfiledItems('dashboard')
         actions.loadUnfiledItems('notebook')
+
+        // Restore saved state
+        const savedExpanded = localStorage.getItem('posthog_project_tree_expanded')
+        const savedActive = localStorage.getItem('posthog_project_tree_active')
+        const savedPath = localStorage.getItem('posthog_project_tree_path')
```

Comment:
I'm not sure we want to have three states to worry about --> one in the database, one in localstorage, and one unsaved in the browser. Just two should be enough --> skip the localstorage?